ablator/config/main.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, *args, add_attributes=False, **kwargs):
             )
         missing_vals = []
         for k, annotation in self.annotations.items():
-            if not annotation.optional and annotation.state not in [Derived, Stateless]:
+            if not annotation.optional and annotation.state not in [Derived]:
```

Comment:
is there a test for this change?

Reply:
yes here https://github.com/fostiropoulos/ablator/blob/07e43eed45bb12d683471856c25b907af9da10d6/tests/configs/test_config.py#L84